package.json

```diff
@@ -20,16 +20,16 @@
     "sitemap": "^3.0.0",
     "tailwindcss": "1.4.6",
     "v-tooltip": "2.1.3",
-    "vue-server-renderer": "2.6.14",
     "vue-fragment": "1.5.2",
     "vue-loader": "^14.0.0",
+    "vue-server-renderer": "2.6.14",
```

Comment:
Let's not change these. It's causing the build to fail. Also, what are the new build and serve commands? Can we use the existing `docs:dev` command?